src/registry.js

```diff
@@ -28,14 +28,24 @@ module.exports = function Registry(defaultBus) {
   };
 
   // TODO: extendable handlers by other attributes, whether elem is block or not, etc.
-  me.busFor = function(elem) {
-    assert(elem.nodeType === 1);
-    var tag = elem.tagName.toUpperCase();
-
-    return (
-      elem[PROP]
-      || tagHandlers[tag]
-      || defaultBus
-      );
+  me.busFleetFor = function(elem) {
```

Comment:
make this an inner function only - no me.busFleetFor = ...

change type to node

document explicitly that if it's a text node, it will do nothing, for convenience - it generously accepts text nodes